LLReactiveMatchers/Helpers/LLSignalTestRecorder.m

```diff
@@ -169,13 +170,13 @@ - (BOOL) haveFinished {
 }
 
 - (NSSet *)operatingThreads {
-  @synchronized(self) {
-    return [self.activeThreadsInReceivedEvents copy];
-  }
+    @synchronized(self) {
+        return [self.activeThreadsInReceivedEvents copy];
+    }
 }
 
 - (NSUInteger) operatingThreadsCount {
```

Comment:
any reason for the space after ) and before the method name on some of the methods in this file?